designs/vcenter-template-import.md

```diff
@@ -16,18 +16,27 @@ As an EKS Anywhere user:
 * I want to specify the template I want to import in a simple manner
 * I want the create command to know to use the pre-imported template
 
+### Users
+
+There are two types of users who would use this functionality. 
+
+* EKS-A maintainers who are in charge of creating the cluster
```

Comment:
nit: EKS-A users or maybe EKS-A cluster admins
Not big deal, but EKS-A maintainers sounds more like it's us :)

Reply:
Good point